ultralytics/nn/tasks.py

```diff
@@ -427,8 +428,7 @@ def parse_model(d, ch, verbose=True):  # model_dict, input_channels(3)
     for i, (f, n, m, args) in enumerate(d['backbone'] + d['head']):  # from, number, module, args
         m = eval(m) if isinstance(m, str) else m  # eval strings
         for j, a in enumerate(args):
-            with contextlib.suppress(NameError):
-                args[j] = eval(a) if isinstance(a, str) else a  # eval strings
+            args[j] = eval(a) if isinstance(a, str) and (keyword.iskeyword(a) or a in locals()) else a  # eval strings
```

Comment:
Why `iskeyword`? What's the keyword you'd expect to be evaluating? 🤔

Reply:
I used it for 'None' value in yaml files.

Reply:
If that's the only one (and maybe True and False), then I guess this could be

```suggestion
            args[j] = (locals()[a] if a in locals() else ast.literal_eval(a)) if isinstance(a, str) else a  # eval strings
```
so as to avoid using the full, dreadfully unsafe `eval()`... though this would need to be tested out with real data.

Reply:
What a safe method using literal_eval! 

@glenn-jocher 
What do you think of this PR?

Reply:
@developer0hye @akx thanks guys! I think these are both improvements, especially from the security perspective, which I've been working on improving recently. 

I'll try the eval removal version suggested by @akx and see what the CI shows. All model yamls in the repo are loaded and tested during CI, so this will let us know if there are any issues.